git_machete/generated_docs.py

```diff
@@ -0,0 +1,938 @@
+
+        <b>Environment variables:</b>
+           `GITHUB_TOKEN`
+              GitHub API token.
+   """,
+    "clean": """
+        <b>Usage:</b><b>
+          git machete clean [-c|--checkout-my-github-prs] [-y|--yes]</b>
+
+        Synchronizes with the remote repository:
+        * if invoked with `-H` or `--checkout-my-github-prs`, checks out open PRs for the current user associated with the GitHub token and also traverses the chain of pull requests upwards, adding branches one by one to git-machete and checks them out locally as well,
+        * deletes unmanaged branches,
+        * deletes untracked managed branches that have no downstream branch.
+
+        No branch will be deleted unless explicitly confirmed by the user (or unless `-y/--yes` option is passed).
+        Equivalent of `git machete github sync` if invoked with `-H` or `--checkout-my-github-prs`.Note
+
+        To allow GitHub API access for private repositories (and also to perform side-effecting actions like opening a PR, even in case of public repositories),
+        a GitHub API token with `repo` scope is required, see https://github.com/settings/tokens. This will be resolved from the first of:
+        * `GITHUB_TOKEN` env var,
+        * content of the `.github-token` file in the home directory (`~`),
+        * current auth token from the `gh` GitHub CLI,
+        * current auth token from the `hub` GitHub CLI.Note
+
+        GitHub API server URL will be inferred from `git remote`.
+        You can override this by setting the following git config keys:
+           Remote name
+              E.g. `machete.github.remote` = `origin`
+           Organization name
+              E.g. `machete.github.organization` = `VirtusLab`
+           Repository name
+              E.g. `machete.github.repository` = `git-machete`
+
+        To do this, run `git config --local --edit` and add the following section:
+        <dim>
+          [machete "github"]
+              organization = <organization_name>
+              repository = <repo_name>
+              remote = <remote_name>
+        </dim>
+
+        <b>Options:</b>
+           <b>-c</b>, <b>--checkout-my-github-prs</b>
+              Checkout your open PRs into local branches.
+           <b>-y</b>, <b>--yes</b>
+              Don't ask for confirmation when deleting branches from git.
+
+        <b>Environment variables:</b>
+           `GITHUB_TOKEN`
+              GitHub API token.
+   """,
+    "config": """
+        Documentation about available `git machete` config keys and environment variables that change the command's default behavior.
+
+        Note: `config` is not a command as such, just a help topic (there is no `git machete config` command).
+
+        <b>Config keys:</b>
+           `machete.github.{remote,organization,repository}`:
+               When executing `git machete github <subcommand>` command, GitHub API server URL will be inferred from `git remote`.
+
+              Note
```

Comment:
Such loose _Note_s look somewhat weird:

![image](https://user-images.githubusercontent.com/3383210/191567419-24319ef6-ae4e-44ff-b56c-ed02fc6a5e03.png)

I'd rather get rid of them in Python docs